api/controllers/web/conversation.py

```diff
@@ -49,7 +49,7 @@ def delete(self, app_model, end_user, c_id):
 
         conversation_id = str(c_id)
         try:
-            ConversationService.delete(app_model, conversation_id, end_user)
+            ConversationService.delete(app_model, conversation_id)
```

Comment:
Web API can't remove the user's filtering criteria because web API is publicly accessible. If a user somehow obtains the id of other conversations, they can delete someone else's conversation, which is an abuse of authority. 😕